spec/helpers/service_helper/textual_summary_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe ServiceHelper::TextualSummary do
+  context ".textual_orchestration_stack" do
+    before do
+      login_as @user = FactoryGirl.create(:user)
+    end
+
+    subject { textual_orchestration_stack }
+    it 'contains the link to the associated stack' do
+      @os_cloud = FactoryGirl.create(:orchestration_stack_cloud, :name => "cloudstack1")
```

Comment:
Using instance variables here doesn't make any sens, could you export this to `let` blocks combined with contexts. Something like this:
```ruby
subject { FactoryGirl.create(cloud, :name => name) }
context "openstack-cloud" do
  let(:cloud) { :orchestration_stack_cloud }
  let(:name) { 'cloudstack1' }
  it 'contains the link to the associated stack' do
    # expect(subject...)
  end
end
context "openstack-infra" do
  let(:cloud) { :orchestration_stack_openstack_infra }
  let(:name) { 'infrastack1' }  
  it 'contains the link to the associated stack' do
    # expect(subject...)
  end
end
# ...
```